x-pack/legacy/plugins/monitoring/public/np_imports/legacy_imports.ts

```diff
@@ -17,7 +17,6 @@ export { StateManagementConfigProvider } from 'ui/state_management/config_provid
 export { AppStateProvider } from 'ui/state_management/app_state';
 // @ts-ignore
 export { EventsProvider } from 'ui/events';
-export { PersistedState } from 'ui/persisted_state';
```

Comment:
@igoristic Can you verify this change?

Reply:
Yeah, this is actually a leftover from https://github.com/elastic/kibana/pull/58324 which I already tested